docs/ui/components/Sidebar/SidebarTitle.tsx

```diff
@@ -7,7 +7,7 @@ type SidebarTitleProps = {
 } & PropsWithChildren;
 
 export const SidebarTitle = ({ children, Icon }: SidebarTitleProps) => (
-  <div className="flex gap-2 items-center relative ml-3 -mr-4 pb-1 select-none">
+  <div className="flex gap-2 items-center relative ml-3 -mr-4 pb-1 select-text">
```

Comment:
```suggestion
  <div className="flex gap-2 items-center relative ml-3 -mr-4 pb-1">
```

We can just remove the class, and if selection is wanted, `auto` seems to be the best option, hich is a default value:
* https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#auto